settings/intellij-idea/formatter.xml

```diff
@@ -5,12 +5,19 @@
       <option name="TAB_SIZE" value="2" />
     </value>
   </option>
-  <option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="99999" />
-  <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="9999" />
+  <option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="5" />
+  <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="3" />
```

Comment:
❌ I am not 100% sure but I think this sets the thresholds for when to use star imports like `java.util.*`. We want to avoid those, so let's leave the numbers as they are.

Reply:
Reverted thresholds back to their original values.

Reply:
Thanks 😃